### ISSUE TITLE
Amazon Verified Permissions Policy Store Schema Incorrect

### ISSUE THREAD

**Issue by nicholasphair** [aws-cloudformation/cfn-lint#4273]
### CloudFormation Lint Version

1.40.3

### What operating system are you using?

Ubuntu, Mac

### Describe the bug

The schema used for `AWS::VerifiedPermissions::PolicyStore` is incorrect wrt. to the `DeletionProtection` property.

cfn-lint produces the following error using the pasted reproduction template:

```bash
$ cfn-lint --template template.json
E3030 {'Mode': 'ENABLED'} is not one of ['DISABLED', 'ENABLED']
template.json:14:9
```

This issue, I believe, stems from a bad schema for the resource. Namely, there is an [extra enum field that should not be there][1].

The latest schema published does not have this enum field.
```
$ wget https://schema.cloudformation.us-east-1.amazonaws.com/CloudformationSchema.zip
$ unzip CloudformationSchema.zip
$ jq .definitions.DeletionProtection aws-verifiedpermissions-policystore.json
{
  "type": "object",
  "properties": {
    "Mode": {
      "$ref": "#/definitions/DeletionMode"
    }
  },
  "required": [
    "Mode"
  ],
  "additionalProperties": false
}
```

I tried `cfn-lint --update-specs` but that did not help.

I also noticed that the [schemas were updated 4 days ago][2] but [`aws-verifiedpermissions-policystore.json`][3] has not changed in 2 months. I don't know how all of the pieces are kept in sync but maybe there was a breakdown in some of the automation? Just a thought. 

Cheers,
Nick

[1]: https://github.com/aws-cloudformation/cfn-lint/blob/5999083466e3a5da5616d8d3a3342e4347b92f0b/src/cfnlint/data/schemas/providers/us_east_1/aws-verifiedpermissions-policystore.json#L14-L17 
[2]: https://github.com/aws-cloudformation/cfn-lint/commit/c067167204c200817bf2e4a09bccfd00896023ed
[3]: https://github.com/aws-cloudformation/cfn-lint/blob/main/src/cfnlint/data/schemas/providers/us_east_1/aws-verifiedpermissions-policystore.json

### Expected behavior

No error.

### Reproduction template

```json
{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Description": "AWS CloudFormation sample template for creating a policy store for Verified Permissions.",
  "Resources": {
    "MyPolicyStore": {
      "Type": "AWS::VerifiedPermissions::PolicyStore",
      "Properties": {
        "Schema": {
          "CedarJson": "{\"PhotoApp\":{\"commonTypes\":{\"PersonType\":{\"type\":\"Record\",\"attributes\":{\"age\":{\"type\":\"Long\"},\"name\":{\"type\":\"String\"}}},\"ContextType\":{\"type\":\"Record\",\"attributes\":{\"ip\":{\"type\":\"Extension\",\"name\":\"ipaddr\",\"required\":false},\"authenticated\":{\"type\":\"Boolean\",\"required\":true}}}},\"entityTypes\":{\"User\":{\"shape\":{\"type\":\"Record\",\"attributes\":{\"userId\":{\"type\":\"String\"},\"personInformation\":{\"type\":\"PersonType\"}}},\"memberOfTypes\":[\"UserGroup\"]},\"UserGroup\":{\"shape\":{\"type\":\"Record\",\"attributes\":{}}},\"Photo\":{\"shape\":{\"type\":\"Record\",\"attributes\":{\"account\":{\"type\":\"Entity\",\"name\":\"Account\",\"required\":true},\"private\":{\"type\":\"Boolean\",\"required\":true}}},\"memberOfTypes\":[\"Album\",\"Account\"]},\"Album\":{\"shape\":{\"type\":\"Record\",\"attributes\":{}}},\"Account\":{\"shape\":{\"type\":\"Record\",\"attributes\":{}}}},\"actions\":{\"viewPhoto\":{\"appliesTo\":{\"principalTypes\":[\"User\",\"UserGroup\"],\"resourceTypes\":[\"Photo\"],\"context\":{\"type\":\"ContextType\"}}},\"createPhoto\":{\"appliesTo\":{\"principalTypes\":[\"User\",\"UserGroup\"],\"resourceTypes\":[\"Photo\"],\"context\":{\"type\":\"ContextType\"}}},\"listPhotos\":{\"appliesTo\":{\"principalTypes\":[\"User\",\"UserGroup\"],\"resourceTypes\":[\"Photo\"],\"context\":{\"type\":\"ContextType\"}}}}}}"
        },
        "ValidationSettings": {
          "Mode": "STRICT"
        },
        "DeletionProtection": {
          "Mode": "ENABLED"
        }
      }
    }
  }
}
```
